build.py

```diff
@@ -13,9 +13,7 @@
     for settings, options, env_vars, build_requires, reference in builder.items:
         installers = ["nasm_installer/2.13.02@bincrafters/stable"]
         if build_shared.get_os() == "Windows":
-            installers.append("cygwin_installer/2.9.0@bincrafters/stable")
-            if os.getenv('MINGW_CONFIGURATIONS', ''):
-                installers.append("mingw_installer/1.0@conan/stable")
+            installers.append("msys2/20161025")
```

Comment:
This looks completely unnecessary as msys2 is already as a build requirement in the conanfile